src/store/slices/users/fetchUsers.ts

```diff
@@ -19,3 +31,24 @@ export const fetchData = createAsyncThunk(
     return data;
   }
 );
+
+export const fetchSignedInUSer = createAsyncThunk(
```

Comment:
E bine ca le-ai creat dar unde sunt folosite? unde e slice-u de redux?

Reply:
Done! Bagat in slice-ul de redux.